src/services/edusharing/index.js

```diff
@@ -35,21 +46,27 @@ class MerlinToken {
 
 module.exports = (app) => {
 	const eduSharingRoute = '/edu-sharing';
-	const merlinRoute = `${eduSharingRoute}/merlinToken`;
-	const docRoute = `${eduSharingRoute}/api`;
+	const eduSharingPlayerRoute = '/edu-sharing/player';
+	const merlinRoute = '/edu-sharing/merlinToken';
+	const docRoute = '/edu-sharing/api';
 
-	app.use(docRoute, staticContent(path.join(__dirname, '/docs/openapi.yaml')));
+	app.use(eduSharingRoute, new EduSharing(), (req, res) => {
+		res.send(res.data);
```

Comment:
I do not understand the to express middlewares for res.send why you want to cut the existing pipline in app.js?
Line 60 too.

Reply:
Yes but for what is this needed. It skip to many of our functionallity. I think the implementation you has as base is written in a bachlor side project before the project start. It is not really a argumentation base. I want to know for what you need this, otherwise the code should be only include:

app.use(eduSharingRoute, new EduSharing());

Reply:
Done